src/olympia/scanners/migrations/0042_remove_waffle_switch.py

```diff
@@ -0,0 +1,15 @@
+# Generated by Django 2.2.12 on 2021-02-01 16:11
+from django.db import migrations
+
+
+def remove_waffle_switch(apps, schema_editor):
+    Switch = apps.get_model('waffle', 'Switch')
+
+    Switch.objects.filter(name='enable-scanner-results-api').delete()
```

Comment:
This is not the switch we want to remove.

Reply:
my bad! the name supposed to be yara-read-binary, right?

Reply:
Yes

Reply:
Modified now